Ports/ssmtp/package.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='ssmtp'
+version='2.64-11'
+files="https://salsa.debian.org/debian/ssmtp/-/archive/debian/${version}/ssmtp-debian-${version}.tar.gz ssmtp-debian-${version}.tar.gz 82abe0fb54e2ee209e9f8745498ce5f4e35f2f2b1bf95bd5e6dfbcfc61a8ebe2"
+auth_type='sha256'
+useconfigure=true
+workdir="ssmtp-debian-${version}"
+configopts=('--enable-ssl' '--enable-md5auth' "--prefix=${SERENITY_INSTALL_ROOT}/usr/local")
+depends=('openssl')
+
+pre_patch() {
+    # Debian released multiple patches that fix issues with ssmtp. But they also decided to replace openssl with gnutls for internal licencing reasons.
+    # As we have a stable openssl port and no stable gnutls port, we skip that patch.
+    run sed -i "/01-374327-use-gnutls.patch/d" debian/patches/series
+    # We will also skip the solaris patch as it messes with `generate_config` we already messed with :^)
+    run sed -i "/02-557725-solaris.patch/d" debian/patches/series
```

Comment:
These `sed -i`s fail on MacOS, would it be possible to also use `run_replace_in_file` here?

Reply:
No, because than it would fail on the newline. I will look into the `perl` syntax to delete a line, should be similar.